composer.json

```diff
@@ -12,14 +12,14 @@
     }
   ],
   "require": {
-    "php": ">=7.1",
+    "php": ">=7.4",
     "ext-curl": "*",
     "guzzlehttp/guzzle": "~6.3",
-    "symfony/config" : "~4.3||~5.0",
-    "symfony/dependency-injection": "~3.4||~4.3||~5.0",
-    "symfony/event-dispatcher": "~4.3||~5.0",
-    "symfony/http-kernel": "~3.4||~4.3||~5.0",
-    "symfony/yaml" : "~3.4||~4.3||~5.0"
+    "symfony/config" : "~4.3||~5.0||~6.0",
+    "symfony/dependency-injection": "~3.4||~4.3||~5.0||~6.0",
+    "symfony/event-dispatcher": "~4.3||~5.0||~6.0",
+    "symfony/http-kernel": "~3.4||~4.3||~5.0||~6.0",
+    "symfony/yaml" : "~3.4||~4.3||~5.0||~6.0"
```

Comment:
since this will be a new major version (because we drop support for php 7.1, 7.2, 7.3),
I'd suggest to drop support for very old symfony versions too at the same time (something like `~4.4||~5.0||~6.0`  for all symfony dependencies should be good enough IMO)

but I'm not the official maintainer of this bundle so it's also fine for me if we keep supporting very old symfony versions :)

Reply:
I agree to support only Symfony >= 4.4 (as 4.4 is the lower maintained version).